gitForm.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Github form</title>
+        <meta charset="UTF-8">
+        <style>
+            body{
+                font-family: sans-serif;
+            }
+            .formInputGuideline{
+                color: gray;
+                font-size: 13px;
+                font-weight: 550;
+                opacity: 0.7;
+            }
+            input{
+                border-radius: 5px;
+                border: 1.5px solid rgba(200,200,200,0.8);
+                outline: none;
+                padding-top: 8px;
+                padding-bottom: 8px;
+                padding-left: 6px;
+                width: 280px;
+                font-size: 15px;
+                font-weight: 500;
+                opacity: 0.7;
+                margin-bottom: 6px;
+            }
+            ::placeholder{
+                opacity: 0.85;
+            }
+            form{
+                display: block;
+                overflow: hidden;
+                border: 1px solid rgba(200,200,200,0.8); 
+                border-radius: 5px;
+                padding: 10px;
+                width: 300px;
+                height: 450px;
+            }
+            .descriptionPart{
+                color: gray;
+                font-size: 12px;
+                font-weight: 500;
+                margin: 3px;
+            }
+            .Two{
+                text-align: center;
+            }
+            a{
+                text-decoration: none;
+                color: #55A9D4;
+            }
+            button{
+                margin: 4px;
+                padding-top: 25px; 
+                padding-bottom: 25px;
+                padding-left: 85px;
+                padding-right: 85px;
```

Comment:
Even we did not work this yes, I am glad you know what `padding-*` properties do. If you wanted top optimize this code, you could rewrite  those 4 lines od CSS into one using `padding: top right bottom left` or in your case `padding: 25px 85px;` (under the hood - 25px 85px gets extended as it was written 25px 85px 25px 85px.

Looks nice anyways!

Reply:
I completely forgot about this. Much cleaner indeed!